### PR TITLE
Add GH build badge to README

### PR DESCRIPTION
**Pull request by gliptak**

### CODE REVIEW

**Comment by gliptak** [comment]
@rogergzousc @Snapchat please review

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # AWS Support Cases Aggregator
 _AWS Support Cases aggregation for a multi-account organization_
 
+[![ci](https://github.com/sSnapchat/aws-support-tickets-aggregator/ci/badge.svg?branch=master)](https://github.com/Snapchat/aws-support-tickets-aggregator/actions?query=workflow%3Aci+branch%3Amaster)
```
Comment (rogergzousc) [inline README.md]: I think the badge link here isn't correct. Should be Snapchat, not sSnapchat.

Comment (gliptak) [reply README.md]: corrected

File: README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # AWS Support Cases Aggregator
 _AWS Support Cases aggregation for a multi-account organization_
 
+[![ci](https://github.com/Snapchat/aws-support-tickets-aggregator/ci/badge.svg?branch=master)](https://github.com/Snapchat/aws-support-tickets-aggregator/actions?query=workflow%3Aci+branch%3Amaster)
```
Comment (rogergzousc) [inline README.md:4]: Apologies for the delay. I'm not very familiar with ci badges on Github.

It does appear that the proper format should include `workflows` in the url, like `https://github.com/Snapchat/aws-support-tickets-aggregator/workflows/ci/badge.svg?branch=master`. I got this URL from going to the URL linked here (`https://github.com/Snapchat/aws-support-tickets-aggregator/actions?query=workflow%3Aci+branch%3Amaster`) and copying the text from "Create status badge".

Please amend the badge image URL.

Comment (gliptak) [reply README.md]: @rogergzousc thank you for looking into this. updated

**Pull request merged**